src/pay/batch_payer.py

```diff
@@ -393,37 +393,39 @@ def attempt_single_batch(self, payment_records, op_counter, dry_run=None):
             if payment_item.paymentaddress.startswith('KT'):
                 simulation_status, simulation_results = self.simulate_single_operation(payment_item, pymnt_amnt, branch, chain_id)
                 if simulation_status == PaymentStatus.FAIL:
-                    logger.info("Payment to {} script could not be processed (Possible reason: liquidated contract). Skipping. (think about redirecting the payout to the owner address using the maps rules. Please refer to the TRD documentation or to one of the TRD maintainers)"
+                    logger.info("Payment to {} script could not be processed. Possible reason: liquidated contract. Skipping. Think about redirecting the payout to the owner address using the maps rules. Please refer to the TRD documentation or to one of the TRD maintainers."
                                 .format(payment_item.paymentaddress))
                     payment_item.paid = PaymentStatus.AVOIDED
                     continue
                 gas_limit, tx_fee, storage_limit = simulation_results
                 burn_fee = COST_PER_BYTE * storage_limit
                 total_fee = tx_fee + burn_fee
-                # Bound the total (baker and burn) fee by the reward amount in case of KT1 accounts
+
                 if total_fee > FEE_LIMIT_CONTRACTS:
-                    logger.info("Payment to {:s} script requires higher fees than reward amount. Skipping. (Needed fee: {:10.6f} XTZ, Max fee: {:10.6f} XTZ) Either configure a higher fee or redirect to the owner address using the maps rules. Refer to the TRD documentation."
+                    logger.info("Payment to {:s} script requires higher fees than reward amount. Skipping. Needed fee: {:10.6f} XTZ, max fee: {:10.6f} XTZ. Either configure a higher fee or redirect to the owner address using the maps rules. Refer to the TRD documentation."
                                 .format(payment_item.paymentaddress, total_fee / MUTEZ, FEE_LIMIT_CONTRACTS / MUTEZ))
                     payment_item.paid = PaymentStatus.AVOIDED
                     continue
 
-                # Fees within safety limits; Subtract from payment amount
+                # Subtract burn fee from the payment amount
                 orig_pymnt_amnt = pymnt_amnt
-                pymnt_amnt = max(pymnt_amnt - total_fee, 0)  # ensure not less than 0
-                logger.info("Payment to {} script requires {:.0f} gas * {:.2f} mutez-per-gas + {:10.6f} default fee = {:10.6f} total fees; Payment reduced from {:10.6f} to {:10.6f}".format(
-                            payment_item.paymentaddress, gas_limit, MUTEZ_PER_GAS_UNIT, self.default_fee / MUTEZ, total_fee / MUTEZ, orig_pymnt_amnt / MUTEZ, pymnt_amnt / MUTEZ))
+                pymnt_amnt = max(pymnt_amnt - burn_fee, 0)  # ensure not less than 0
+                logger.info("Payment to {} script requires {:.0f} gas * {:.2f} mutez-per-gas + {:10.6f} burn fee; Payment reduced from {:10.6f} to {:10.6f}".format(
```

Comment:
The printed info is not consistent with the code: only the burn fee is substructed in this case.